File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntDeployPaymentProviderMap.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdPaymentProvider;

import org.jetbrains.annotations.Nullable;

import java.util.Map;

@SuppressWarnings("unused")
public class StudioEntDeployPaymentProviderMap extends StudioBase
{
  @Nullable
  public MetaIdPaymentProvider defaultPaymentProviderId;

  public MetaIdPaymentProvider[] keys;

  public Map<MetaIdPaymentProvider, StudioEntPaymentProvider> map;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntDeployPaymentProviderMap.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdPaymentProvider
import com.neome.api.meta.base.dto.StudioBase
import com.neome.api.meta.base.dto.StudioEntPaymentProvider

interface StudioEntDeployPaymentProviderMap : StudioBase
{
  val defaultPaymentProviderId: MetaIdPaymentProvider?
  val keys: List<MetaIdPaymentProvider>
  val map: Map<MetaIdPaymentProvider, StudioEntPaymentProvider>
}